### DISCUSSION TITLE
print a checkbox checked

### DISCUSSION THREAD

**Discussion by franfrLor** [quasarframework/quasar#10007] [General - Components / Directives / etc] [answered]
how we can print a checkbox checked ?
I want when I use the print function of the browser the box appears checked on the printout

https://codepen.io/franfrlor/pen/bGWeeLV?editors=101
![checkbox](https://user-images.githubusercontent.com/64161955/124912153-e6f52200-dfed-11eb-9025-e4e72f23f486.jpg)

**Comment by ontwikkelfabriek** [comment]
@franfrLor 

> https://codepen.io/pen/?&editors=101

Your codepen.io link is **not** ok, it's a default url and **empty**.

Did you see the Quasar's print classes:
https://quasar.dev/style/visibility#printing-related

> how we can print a checkbox checked ?

Also you can try these vue print libraries:
https://www.npmjs.com/package/vue-print-nb
https://github.com/mycurelabs/vue-html-to-paper

**Reply by franfrLor** [reply]
I updated the link.
I don't see how print libraries can help me?
When I print a checked box, it appears unchecked

**Comment by justinbeaty** [comment, marked as answer]
@franfrLor This seems like weird chrome behavior, but I also learned something new today. Try adding this CSS to your codepen:

```
.q-checkbox__inner--indet .q-checkbox__bg, .q-checkbox__inner--truthy .q-checkbox__bg {
    -webkit-print-color-adjust: exact;
}
```

**Edit:** I say it's weird, because normally chrome doesn't print background colors unless a box is checked, but even when I did check that box, it was still blank. That could have something to do with the codepen / iframe though. Still, I now know how to force specific elements to print their bg color even without that box checked.

**Reply by franfrLor** [reply]
Thank you very much

**Reply by ontwikkelfabriek** [reply]
@justinbeaty 

Good to see your back posting! Welcome back Beets!

**Reply by justinbeaty** [reply]
@ontwikkelfabriek Thanks, I finally feel like I have some free time again to help!
